lte/gateway/c/core/oai/tasks/gtpv1-u/gtp_tunnel_openflow.c

```diff
@@ -300,6 +300,10 @@ int openflow_add_tunnel(
     struct ip_flow_dl* flow_dl, uint32_t flow_precedence_dl, char* apn) {
   uint32_t gtp_portno = find_gtp_port_no(enb, enb_ipv6, false);
 
+  if (spgw_config.sgw_config.ebpf_enabled) {
+    update_ebpf_dl_map(ue, enb, o_tei);
+  }
```

Comment:
need to check if this is IPv6 only flow or not.

Reply:
added